src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.util.Random;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class Duke {
```

Comment:
Generally, you could consider adding more comments to describe what your methods or classes are doing.